askama_shared/src/generator.rs

```diff
@@ -1184,7 +1184,8 @@ impl<'a, S: std::hash::BuildHasher> Generator<'a, S> {
             return Ok(DisplayWrap::Unwrapped);
         }
 
-        if name == "escape" || name == "safe" || name == "e" || name == "json" {
+        const FILTERS: [&str; 5] = ["safe", "escape", "e", "json", "yaml"];
+        if FILTERS.contains(&name) {
```

Comment:
or did you want it in `filters`, like with `filters::BUILT_IN_FILTERS`?

Reply:
Nah, I think this is fine for now.